spec/features/cases/offender_sar/generate_acknowledgement_letter_spec.rb

```diff
@@ -25,8 +25,8 @@
       expect(cases_show_page).to have_content "Send acknowledgement letter"
       click_on "Send acknowledgement letter"
 
+      sleep(5)
```

Comment:
Five seconds is quite long for a sleep on a browser test - is 5 needed or could it be 1 or 2 seconds instead?

Reply:
@willmcb  this is dummy change, I will remove it later,  this error is really random error :(,  happened a few times. But I found out such random error will be "cached" even you try to run the whole test again, so if you just put some dummy change, it may pass for next time,  then you can remove the dummy change afterwards, it most cases, it will still pass. I knew it is tricky 🤦